ext/sockets/conversions.c

```diff
@@ -1454,7 +1454,7 @@ void *from_zval_run_conversions(const zval			*container,
 
 	structure = ecalloc(1, struct_size);
 
-	zend_llist_add_element(&ctx.keys, &top_name);
+	zend_llist_add_element(&ctx.keys, (void *) &top_name);
```

Comment:
You can change the zend_llist_add_element signature to accept a const void * instead.

Reply:
Ah, indeed. Thanks!

Reply:
You might also want to do this for prepend_element for the sake of consistency.

Reply:
Makes sense.